include/matx/transforms/reduce.h

```diff
@@ -2234,6 +2234,52 @@ void __MATX_INLINE__ argmin_impl(OutType dest, TensorIndexType &idest, const InT
 #endif
 }
 
+/**
+ * Compute min and max reduction of a tensor and returns value + index
```

Comment:
Of an operator

Reply:
Fixed for argminmax, argmax, and argmin.